src/test/java/se/sundsvall/billingdatacollector/service/scheduling/fallout/FalloutJobHandlerTest.java

```diff
@@ -0,0 +1,120 @@
+package se.sundsvall.billingdatacollector.service.scheduling.fallout;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoInteractions;
+import static org.mockito.Mockito.verifyNoMoreInteractions;
+import static org.mockito.Mockito.when;
+
+import java.util.List;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import se.sundsvall.billingdatacollector.integration.db.model.FalloutEntity;
+import se.sundsvall.billingdatacollector.integration.messaging.FalloutMessageProperties;
+import se.sundsvall.billingdatacollector.integration.messaging.MessagingClient;
+import se.sundsvall.billingdatacollector.service.DbService;
+
+import generated.se.sundsvall.messaging.EmailBatchRequest;
+import generated.se.sundsvall.messaging.MessageBatchResult;
+
+@ExtendWith(MockitoExtension.class)
+class FalloutJobHandlerTest {
+
+	@Mock
+	private MessagingClient mockMessagingClient;
+
+	@Mock
+	private FalloutMessageProperties mockProperties;
+
+	@Mock
+	private FalloutMessageProperties.FalloutMailTemplate mockFalloutMailTemplate;
+
+	@Mock
+	private MessagingFalloutMapper mockFalloutMapper;
+
+	@Mock
+	private DbService mockDbService;
+
+	private FalloutJobHandler jobHandler;
+
+	@BeforeEach
+	void setUp() {
+		jobHandler = new FalloutJobHandler(mockMessagingClient, mockProperties, mockFalloutMapper, mockDbService);
```

Comment:
Replace this method with `@InjectMocks` ?

Reply:
I injected string values into it before, but they've been moved to MessagingFalloutMapper, forgot to change it. Will fix.